fog/view/protocol/src/user_rng_set.rs

```diff
@@ -172,6 +172,8 @@ impl From<DecodeError> for TxOutRecoveryError {
     }
 }
 
+// Is this error type ever used?
+#[allow(dead_code)]
```

Comment:
`dead_code`: is `RngSetError` used?

Reply:
it doesn't look like this one is used. It should be removed